docs/controls/TabbedCommandBar.md

```diff
@@ -0,0 +1,248 @@
+      <AppBarButton Label="3D Models">
+        <AppBarButton.Icon>
+          <FontIcon FontFamily="Segoe MDL2 Assets" Glyph="&#xF158;" />
+        </AppBarButton.Icon>
+      </AppBarButton>
+      <AppBarSeparator/>
+      <AppBarButton Label="Add-ins">
+        <AppBarButton.Icon>
+          <FontIcon FontFamily="Segoe MDL2 Assets" Glyph="&#xECAA;" />
+        </AppBarButton.Icon>
+      </AppBarButton>
+      <controls:TabbedCommandBarItem.SecondaryCommands>
+        <AppBarButton Icon="Add" Label="New item" />
+      </controls:TabbedCommandBarItem.SecondaryCommands>
+    </controls:TabbedCommandBarItem>
+    <controls:TabbedCommandBarItem Header="Shape Format" IsContextual="True" Visibility="{Binding ElementName=ContextualToggle, Path=IsOn}">
+      <AppBarButton Label="Shape Fill">
+        <AppBarButton.Icon>
+          <FontIcon FontFamily="{ThemeResource SymbolThemeFontFamily}" Glyph="&#xE771;" />
+        </AppBarButton.Icon>
+      </AppBarButton>
+      <AppBarButton Label="Shape Outline">
+        <AppBarButton.Icon>
+          <FontIcon FontFamily="{ThemeResource SymbolThemeFontFamily}" Glyph="&#xE76D;" />
+        </AppBarButton.Icon>
+      </AppBarButton>
+    </controls:TabbedCommandBarItem>
+  </controls:TabbedCommandBar.MenuItems>
+</controls:TabbedCommandBar>
+```
+
+## Sample Output
+
+ ![TabbedCommandBar Overview](../resources/images/Controls/TabbedCommandBar.gif)
+
+## Properties
+
+### TabbedCommandBar Properties
+
+The TabbedCommandBar does not add any of its own properties. See [NavigationView](https://docs.microsoft.com/uwp/api/windows.ui.xaml.controls.navigationview#properties) for a list of accessible properties.
+
+> [!IMPORTANT]
+> Do not use `ItemsStackPanel` if you override the ItemsPanel.  It is suggested to keep the `TabWidthBehavior` to `Actual` when using a custom panel.
+
+### TabbedCommandBarItem Properties
+
+| Property | Type | Description |
+| -- | -- | -- |
+| Header | object | Header Content of the Tab. |
+| IsContextual | bool | Set to true for the TabbedCommandBarItem to be styled as a contextual Tab. |
+| OverflowButtonAlignment | HorizontalAlignment | Indicates the alignment of the command overflow button. |
+| CommandAlignment | HorizontalAlignment | Indicates the alignment of this Tab's commands. |
+
+## Events
+
+### TabbedCommandBar Events
+
+| Events | Description |
+| -- | -- |
+| SelectionChanged | Fires when a different Tab is selected. |
+
+## Examples
+
+The following setup demos contextual tabs, and binding to their visibility:
+
+```xaml
+<controls:TabbedCommandBar>
+  <controls:TabbedCommandBar.MenuItems>
+    <controls:TabbedCommandBarItem Header="Home">
+      <AppBarButton Icon="Undo" Label="Undo"/>
+      <AppBarButton Icon="Redo" Label="Redo"/>
+      <AppBarButton Icon="Paste" Label="Paste"/>
+    </controls:TabbedCommandBarItem>
+    <controls:TabbedCommandBarItem Header="View" IsContextual="True" Visibility="{Binding ElementName=ContextualToggle, Path=IsOn}">
```

Comment:
This Binding didn't work this way in the Sample App, does it work if you use x:Bind in a stand-alone app? Want to make sure we're providing working code here as a sample.

Reply:
Whoops, yeah, I think I copied that from the sample before we (you) figured out it didn't work like that.